easybuild/easyconfigs/o/OpenSSL/OpenSSL-1.1.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'EB_OpenSSL_wrapper'
+
+name = 'OpenSSL'
+version = '1.1'
+
+homepage = 'https://www.openssl.org/'
+description = """The OpenSSL Project is a collaborative effort to develop a robust, commercial-grade, full-featured,
+ and Open Source toolchain implementing the Secure Sockets Layer (SSL v2/v3) and Transport Layer Security (TLS v1) 
+ protocols as well as a full-strength general purpose cryptography library. """
+
+toolchain = SYSTEM
+toolchainopts = {'pic': True}
+
+osdependencies = [
+    'perl',
+    ('zlib-devel', 'zlib1g-dev'),
```

Comment:
Are you sure this is required?
Installation works fine of Ubuntu 18.04 without having devel package for `zlib` (or even `zlib` itself) installed.

Reply:
@boegel you are right, the headers are not needed. Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12858/commits/f3385c706220f3fe35734570500a638e3a3623a5